tools/compiletest/src/runtest.rs

```diff
@@ -395,7 +395,12 @@ impl<'test> TestCx<'test> {
     /// the expected output in `expected` file.
```

Comment:
The doc for this function becomes outdated with this change.

Reply:
Good catch. Has been updated!